docs/api/browser-window.md

```diff
@@ -457,16 +457,19 @@ height areas you have within the overall content view.
 * `width` Integer
 * `height` Integer
 
+`animate` Boolean (optional) _OS X_
```

Comment:
@zcbenz but the list describes the `options` object, and the `animate` parameter is not in the `options` object.